integrations/nlu-rasa/src/RasaNlu.ts

```diff
@@ -3,17 +3,30 @@ import {
   AxiosRequestConfig,
   AxiosResponse,
   DeepPartial,
+  EntityMap,
   HandleRequest,
   Jovo,
   NluData,
   NluPlugin,
   PluginConfig,
 } from '@jovotech/framework';
-import { RasaResponse } from './interfaces';
+import { RasaEntity, RasaIntent, RasaResponse } from './interfaces';
 
 export interface RasaNluConfig extends PluginConfig {
-  serverUrl: string;
-  serverPath: string;
+  serverUrl?: string;
```

Comment:
`serverUrl` and `serverPath` should not be optional here due to being required to call the Rasa server.

Reply:
Done. thought i did that already